-1

Tested at: ac996786f6b8cf255b882413be4d792f452d63ce

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-18427/19307/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/TauAnalysis/MCEmbeddingTools/plugins/TrackMergeremb.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/TauAnalysis/MCEmbeddingTools/plugins/EmbeddingVertexCorrector.cc 
>> Compiling edm plugin /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/TauAnalysis/MCEmbeddingTools/plugins/MuMuForEmbeddingSelector.cc 
In file included from /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/GeneratorInterface/Pythia6Interface/plugins/Pythia6HadronizerFilter.cc:3:0:
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/GeneratorInterface/Core/interface/HadronizerFilter.h: In constructor 'edm::HadronizerFilter<HAD, DEC>::HadronizerFilter(const edm::ParameterSet&)':
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/GeneratorInterface/Core/interface/HadronizerFilter.h:164:33: error: 'Transtion' is not a member of 'edm'
     produces<GenLumiInfoHeader, edm::Transtion::BeginLuminosityBlock>();
                                 ^
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/GeneratorInterface/Core/interface/HadronizerFilter.h:164:33: error: 'Transtion' is not a member of 'edm'
/build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_1_X_2017-04-20-1100/src/GeneratorInterface/Core/interface/HadronizerFilter.h:164:5: error: parse error in template argument list
     produces<GenLumiInfoHeader, edm::Transtion::BeginLuminosityBlock>();
</pre>

--- a/GeneratorInterface/Core/interface/GeneratorFilter.h
+++ b/GeneratorInterface/Core/interface/GeneratorFilter.h
@@ -114,7 +114,7 @@ namespace edm
 
     produces<edm::HepMCProduct>("unsmeared");
     produces<GenEventInfoProduct>();
-    produces<GenLumiInfoHeader, edm::Transtion::BeginLuminosityBlock>();
+    produces<GenLumiInfoHeader, edm::Transition::BeginLuminosityBlock>();
     produces<GenLumiInfoProduct, edm::Transition::EndLuminosityBlock>();
     produces<GenRunInfoProduct, edm::Transition::EndRun>();
  

--- a/GeneratorInterface/Core/interface/HadronizerFilter.h
+++ b/GeneratorInterface/Core/interface/HadronizerFilter.h
@@ -161,7 +161,7 @@ namespace edm
 
     produces<edm::HepMCProduct>("unsmeared");
     produces<GenEventInfoProduct>();
-    produces<GenLumiInfoHeader, edm::Transtion::BeginLuminosityBlock>();
+    produces<GenLumiInfoHeader, edm::Transition::BeginLuminosityBlock>();
     produces<GenLumiInfoProduct, edm::Transition::EndLuminosityBlock>();
     produces<GenRunInfoProduct, edm::Transition::EndRun>();
     if(filter_)

